app/views/servers/check_installation.js.erb

```diff
@@ -2,5 +2,20 @@
   $("[data-server-status~=checking]").addClass("hidden");
   $("[data-server-status~=connected]").removeClass("hidden");
 <% else %>
+  <% if @server.install_step == 1 %>
```

Comment:
What do you think about making this integer an `enum`? Then we can do things like:
```ruby
if @server.updating_apt_get?
elsif @server.installing_docker?
elsif @server.installing_dokku?
```